### ISSUE TITLE
QA Ph1_Accessibility: Missing alternative text 

### ISSUE THREAD

**Issue by Remiqa** [GoogleChrome/ioweb2016#198]
There are six cases of missing alternative text.  

Steps to reproduce: 
1. Go to https://phase1-dot-io-webapp-staging.appspot.com/io2016/
2. Using WAVE extension for Chrome add the filter WCAG 2.0 AA and view flags 
3. Note missing alternative text flags - see screenshots for reference 

Chrome 48 Mac OS X 10.11 

![gio_198a](https://cloud.githubusercontent.com/assets/7585604/13297221/13192fe2-dae6-11e5-84b1-e38866bd4a18.png)
![gio_198b](https://cloud.githubusercontent.com/assets/7585604/13297222/133a650e-dae6-11e5-9dbc-7671f5b85a08.png)
